### ISSUE TITLE
[Earth] Train VGG16 on cervix ultrasound only

### ISSUE THREAD

**Issue by Domnyk** [Domnyk/pallas#22]
Assumptions:
* Untrained VGG16
* Cervix ultrasound datasets contains around 350 images
  * Data augmentation is necessary because dataset is very small
  * I have to keep preterm/control ratio same in train/val/test subsets
* Train VGG16 on cervix ultrasound only

Aspects to consider in future
* If performance of network will be poor, I can consider using pretrained network

Expectations
* I expect low AUC (around 0.5) since dataset is very small

**Comment by Domnyk** [comment]
See #23. Closing as I will move to skorch in new issue

**Comment by Domnyk** [comment]
**Results**
One model achieved AUC of 0.77 on test set after 7 epochs. However AUC of val set was below 0.5. Link to model: [click](https://drive.google.com/drive/folders/102DZD08L1PKFOva8RKmvjduxdtRH0RQ6?usp=sharing)

**Next steps**
Use cross validation to bump AUC on val set. See #25
